notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-989_analysis_inertia_compensation.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #64.            hp_measured_forces = await efd_client.select_time_series(
</pre><p>I would encourage you to use the summit utils getEfdData instead of select_time_series, so that we all use the same query format</p><p>Or if you are not going to use getEfdData you dont need to import it in the first cell. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>